config/rmt.yml

```diff
@@ -47,3 +47,6 @@ web_server:
 
 connect_api:
   disable_system_tokens: false
+
+registry:
+  token_expiration: 28800 # 8.hours.to_i
```

Comment:
Could we hardcode the default of 8h in the engine code, and only overwrite when this config value is present? Then we would not need to add this default to the config. (Same reason as above, I think this might confuse users that are using RMT without pubcloud engines (99.9%)).

Reply:
Updated

Reply:
Well the idea of the hard coding was to remove it from the config. At least in my diff in the UI I still see the value in the config.